subworkflows/local/annotation_cache_initialisation/main.nf

```diff
@@ -39,7 +40,8 @@ workflow ANNOTATION_CACHE_INITIALISATION {
 
     if (vep_enabled) {
         def vep_annotation_cache_key = (vep_cache == "s3://annotation-cache/vep_cache/") ? "${vep_cache_version}_${vep_genome}/" : ""
-        def vep_cache_dir = "${vep_annotation_cache_key}${vep_species}/${vep_cache_version}_${vep_genome}"
+        def vep_species_suffix = vep_custom_args.contains("--merged") ? '_merged' : (vep_custom_args.contains("--refseq") ? '_refseq' : '')
+        def vep_cache_dir = "${vep_annotation_cache_key}${vep_species}${vep_species_suffix}/${vep_cache_version}_${vep_genome}"
```

Comment:
Has this change been tested? (I don't suppose we have a CI-test for that, right?)